src/modules/entity-history/stores/current-entity-history-date-store.js

```diff
@@ -1,14 +1,13 @@
 import nuclearJS from 'nuclear-js';
-import dateToStr from '../../../util/date-to-str';
 import actionTypes from '../action-types';
 
 const { Store } = nuclearJS;
 
 const INSTANCE = new Store({
   getInitialState() {
-    const yesterday = new Date();
-    yesterday.setDate(yesterday.getDate() - 1);
```

Comment:
The idea behind this is that graphs don't do well with little data. Looking at yesterday will give us more data.